compat_test.go

```diff
@@ -1,41 +0,0 @@
-package bls
-
-import (
-	"testing"
-
-	oldBls "github.com/drand/bls12-381"
-	"github.com/drand/kyber"
-	"github.com/stretchr/testify/require"
-)
-
-type Hashable interface {
-	Hash([]byte) kyber.Point
-}
-
-func TestCompat(t *testing.T) {
```

Comment:
is this no longer true, or do we no longer care?

Reply:
Sorry forgot to mention: yes I consider we do not care anymore. Travis runs a full compatibility suite with test vectors  from this summer that drand and Filecoin can verify. As long as that's correct, it's all good. This test came from before that test vector.
I removed it because the old library `github.com/drand/bls12-381` is no longer compatible with the latest kyber so to make this PR pass I would have needed yet another change in this other repo...